docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+   ```
+
+{{< /tab >}}
+
+{{% tab name="Sugar Controller" %}}
+
+Uninstall the Eventing Sugar Controller by running:
+
+```bash
+kubectl delete -f {{< artifact repo="eventing" file="eventing-sugar-controller.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="GitHub Source" %}}
+
+Uninstall a single-tenant GitHub source by running:
+
+```bash
+kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-github" file="github.yaml" >}}
+```
+
+Uninstall a multi-tenant GitHub source by running:
+
+```bash
+kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-github" file="mt-github.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="Apache Camel-K Source" %}}
+
+Uninstall the Apache Camel-K Source by running:
+
+```bash
+kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-camel" file="camel.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="Apache Kafka Source" %}}
+
+Uninstall the Apache Kafka source by running:
+
+```bash
+kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-kafka" file="source.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="GCP Sources" %}}
+
+Uninstall the GCP sources by running:
+
+```bash
+kubectl delete -f {{< artifact org="google" repo="knative-gcp" file="cloud-run-events.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="Apache CouchDB Source" %}}
+
+Uninstall the Apache CouchDB source by running:
+
+```bash
+kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-couchdb" file="couchdb.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="VMware Sources and Bindings" %}}
+
+Uninstall the VMware sources and bindings by running:
+
+```bash
+kubectl delete -f {{< artifact org="vmware-tanzu" repo="sources-for-knative" file="release.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+
+### Uninstalling an optional Broker (Eventing) layer
+
+Uninstall a Broker (Eventing) layer, if you installed one:
+
+<!-- This indentation is important for things to render properly. -->
+{{< tabs name="eventing_brokers" default="MT-Channel-based" >}}
+{{% tab name="Apache Kafka Broker" %}}
+
+1. Uninstall the Kafka Broker data plane by running the following command:
+
+    ```bash
+    kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+    ```
+
+1. Uninstall the Kafka controller by running the following command:
+
+    ```bash
+    kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+{{< /tab >}}
+
+{{% tab name="MT-Channel-based" %}}
+
+Uninstall the broker by running:
+
+```bash
+kubectl delete -f {{< artifact repo="eventing" file="mt-channel-broker.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+
+### Uninstalling an optional default Channel (messaging) layer
+
+If you installed a default Channel (messaging) layer, uninstall it:
```

Comment:
I don't think you need the word "default" here -- also, this might read "For each Channel (messaging) layer installed, uninstall it:"

Reply:
Okay, I'll change accordingly.